File: C#/Demo/Demo/Station.cs
namespace Library
{
    public class Station
    {
        private double? distance;
        private double? latitude;
        private double? longitude;
        private int? useCount;
        private string id;
        private string name;
        private int? quality;
        private double? contribution;

        // Constructor
        public Station() { }

        // Properties for each field
        public double? Distance
        {
            get { return distance; }
            set { distance = value; }
        }

        public double? Latitude
        {
            get { return latitude; }
            set { latitude = value; }
        }

        public double? Longitude
        {
            get { return longitude; }
            set { longitude = value; }
        }

        public int? UseCount
        {
            get { return useCount; }
            set { useCount = value; }
        }

        public string Id
        {
            get { return id; }
            set { id = value; }
        }

        public string Name
        {
            get { return name; }
            set { name = value; }
        }

        public int? Quality
        {
            get { return quality; }
            set { quality = value; }
        }

        public double? Contribution
        {
            get { return contribution; }
            set { contribution = value; }
        }
    }

}

File: Swift/Demo/Demo/Station.swift
import Foundation

class Station {
    private var _distance: Double?
    private var _latitude: Double?
    private var _longitude: Double?
    private var _useCount: Int64?
    private var _id: String?
    private var _name: String?
    private var _quality: Int64?
    private var _contribution: Double?

    // Constructor
    init() { }

    init(_Distance: Double?, _Latitude: Double?, _Longitude: Double?, _UseCount: Int64?, _Id: String?, _Name: String?, _Quality: Int64?, _Contribution: Double?) {
       self._distance = _Distance
       self._latitude = _Latitude
       self._longitude = _Longitude
       self._useCount = _UseCount
       self._id = _Id
       self._name = _Name
       self._quality = _Quality
       self._contribution = _Contribution
   }
    
    // Properties for each field
    var _Distance: Double? {
        get { return _distance }
        set { _distance = newValue }
    }

    var _Latitude: Double? {
        get { return _latitude }
        set { _latitude = newValue }
    }

    var _Longitude: Double? {
        get { return _longitude }
        set { _longitude = newValue }
    }

    var _UseCount: Int64? {
        get { return _useCount }
        set { _useCount = newValue }
    }

    var _Id: String? {
        get { return _id }
        set { _id = newValue }
    }

    var _Name: String? {
        get { return _name }
        set { _name = newValue }
    }

    var _Quality: Int64? {
        get { return _quality }
        set { _quality = newValue }
    }

    var _Contribution: Double? {
        get { return _contribution }
        set { _contribution = newValue }
    }
}
